docs/wallets/desktop_wallet.md

```diff
@@ -0,0 +1,217 @@
+# Crypto.org Chain Desktop Wallet 
+
+Crypto.org Chain Desktop Wallet (Beta) supports staking operations, validator selections, and governance voting. Users can also view, send and mint NFTs directly, and much more from the Desktop Wallet. The latest version can be found on the [Desktop Wallet page](https://crypto.org/desktopwallet), and the previous releases are also available on the [release page](https://github.com/crypto-com/chain-desktop-wallet/releases).
+
+### Technical support and enquiries
+Hope this guide gives you a better understanding of the use of Crypto.org Chain Desktop Wallet! 
+
+In case you have any enquiries about using the Desktop Wallet, kindly send a message to the Crypto.org Chain [Discord](https://discord.gg/pahqHz26q4) #desktop-wallet channel or [chain@crypto.org](mailto:chain@crypto.org) for assistance.
+
+## Using Desktop Wallet for the first time
+- **Step 1** - [Download](https://crypto.org/desktopwallet) the latest Crypto.org Chain Desktop Wallet.
+- **Step 2** - Run the Desktop Wallet you just installed and and click "*Get Started*". 
+        <img src="./assets/desktop_wallet/0-2.png" />
+- **Step 3** - Before creating or restoring your wallet, you would need to create an App password to protect your desktop wallet. The password is required to be at least 8 characters, containing a letter, a number, and a special character.
+        <img src="./assets/desktop_wallet/0-3.png" />
+**Remark**: It is recommended to create a stronger and unguessable password. A password labeled as "*safe*" and "*strong*" means that a person would have an extremely low probability to guess it correctly.
+- **Step 4** - Choose between "*Restore Wallet*" and "*Create Wallet*"
```

Comment:
Can link it to `### 1.1 - Create Wallet`  and `### 1.2 - Restore Wallet from Mnemonic Phrases`